lib/mayaUsd/render/vp2RenderDelegate/material.h

```diff
@@ -147,10 +159,11 @@ class HdVP2Material final : public HdMaterial
 
     TfToken _surfaceNetworkToken; //!< Generated token to uniquely identify a material network
 
-    HdVP2ShaderUniquePtr _surfaceShader;    //!< VP2 surface shader instance
-    SdfPath              _surfaceShaderId;  //!< Path of the surface shader
-    HdVP2TextureMap      _textureMap;       //!< Textures used by this material
-    TfTokenVector        _requiredPrimvars; //!< primvars required by this material
+    HdVP2ShaderUniquePtr         _surfaceShader;    //!< VP2 surface shader instance
+    SdfPath                      _surfaceShaderId;  //!< Path of the surface shader
+    static HdVP2GlobalTextureMap _globalTextureMap; //!< Texture in use by all materials in MayaUSD
```

Comment:
With a single global static structure we'd never delete the textures until the plugin unloads or Maya exits. At that time it is often not safe to try to destroy the texture, and textures were held in the cache beyond `file -new`.

This way we get the deletion at the right time.